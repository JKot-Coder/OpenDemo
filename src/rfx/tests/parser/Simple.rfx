// :PARSER

BasicTypes:
{
  String: "test",
  Number: 42,
  Float: 0.25,
  Null: null,
  Boolean: true,
  Array: [test, false, 55, 0.27, null, true],
  Object: {
    Boolean: false,
    String: success
  }
}

