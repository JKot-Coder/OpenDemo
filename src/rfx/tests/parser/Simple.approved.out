SourceOutput:[
    {
      "BasicTypes": {
        "Array": [
          "test",
          "string",
          false,
          55,
          0.27,
          null,
          true
        ],
        "Boolean": true,
        "Float": 0.25,
        "Null": null,
        "Number": 42,
        "Object": {
          "Boolean": false,
          "String": "success"
        },
        "String": "test"
      }
    }
]
Diagnostic:[
]
